src/plugins/console/server/lib/spec_definitions/js/ingest.ts

```diff
@@ -112,7 +108,6 @@ const csvProcessorDefinition = {
   },
 };
 
-// Based on https://www.opensearch.org/guide/en/elasticsearch/reference/master/convert-processor.html
```

Comment:
Can we just fix these links in the comments so we still have the reference to the elastic docs? We could also add them to the `noDocumentation` object in the doc links service to keep track of what remains?

Reply:
+1